### PR TITLE
Create wabbit.yml

### PR DESCRIPTION
**Pull request by lookas1**
wabbit.pl is a new startup offering online meetings, online events, online education platform.

Adds or updates the following companies, events, or universities:
 - company wabbit.pl
 

#### Company updates
 - [ x] I have put the most recent relevant date in the "Last Update" column
 - [ N/A] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post)

#### Event updates
 - [ N/A] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ N/A] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

File: _data/companies/wabbit.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: wabbit.pl
+wfh: Evangelist
```
Comment (drewbrew) [inline _data/companies/wabbit.yml]: Could you explain what Evangelist means here?  Did you mean encouraged?

Comment (lookas1) [reply _data/companies/wabbit.yml]: No, we are a group of people working remotely for more than 10 years now for enterprise companies. We have created a service wabbit.pl for online meetings encouraging people to work and meet remotely. We believe our personal experience of remote work is unmatched to most people's imagination of remote work. We want to lead them through these hard times, give them a service in which they can meet family, run cultural meetings etc. We believe in Online Egagement and remote work is just a part of it.

Comment (drewbrew) [reply _data/companies/wabbit.yml]: Ok, that makes a lot of sense. How about `100% Remote` in parentheses?

```suggestion
wfh: Evangelist (100% remote)
```

Comment (lookas1) [reply _data/companies/wabbit.yml]: @drewbrew, yes it looks ok, I don't seem to have an option to commit that change. Can you do it or should I reopen pull request?

Comment (drewbrew) [reply _data/companies/wabbit.yml]: Ok, I'll take care of it next time I'm at the computer. Thanks!

Comment (drewbrew) [reply _data/companies/wabbit.yml]: It turns out I don't have edit permissions. Do you see a "commit suggestion" button at the bottom of my suggestion?

Comment (lookas1) [reply _data/companies/wabbit.yml]: image

No, I can't commit that suggestion. Please see attached

Comment (drewbrew) [reply _data/companies/wabbit.yml]: Looks like GitHub has lost track of your source repository. I'll take your commit and open up a new PR with my fix on top of it.

**Pull request merged**